lib/assets/summary.html.erb

```diff
@@ -189,6 +189,11 @@
 
               <% last_size = screenshot.screen_size %>
             <% end %>
+            
+            <% if last_size != nil %>
+              </div>
```

Comment:
Yes it's for the `<div class="app-screenshot-row">` div. I *think* my change is the right change.

This block of code is looping through all the screenshots for each device in the current language. It has sorted the screenshots by size to ensure all the screenshots for each device are grouped together.

It starts the row of screenshots and opens the div by checking if it's a new row (line 183). It then loops through all the screenshots for that size until it sees that there is a new row and the screenshot size has changed (line 179) where it normally closes the div. It would then open a new div for the next size of screenshots (line 183 again). This works for all except closing the last row of screenshots because we exit the loop before reaching the close div check (on line 179) because we don't have any screenshots left.

Therefore, we have to have a close div tag outside of the loop to close that final row and open div. I don't actually think I needed the `<% if last_size != nil %>` but just wanted to do some defensive programming for this popular tool.

I believe that if the div was placed where you suggested then we would have many close div tags (one for each screenshot) without the corresponding open div tag (for any screenshots after the first one which would get the open div tag). I'm not entirely sure how the preview HTML file would look but I don't think it would be as expected (sorry can't test that right now as I'm currently away).

Hope that makes sense. Happy to discuss further. Thanks!